fe/fe-core/src/main/java/com/starrocks/http/HttpServerHandler.java

```diff
@@ -84,6 +87,11 @@ public void channelRead(ChannelHandlerContext ctx, Object msg) throws Exception
                 validateRequest(ctx, request);
             } catch (Exception e) {
                 LOG.warn("accept bad request: {}, error: {}", request.uri(), e.getMessage(), e);
+                Iterator headers = request.headers().iteratorAsString();
```

Comment:
How about moving this logic into httprequest and adding a function debugHeader?

Reply:
you mean add an implementation for httprequest? @imay